testsuite/features/core_ubuntu_salt_ssh.feature

```diff
@@ -59,7 +59,7 @@ Feature: Bootstrap a SSH-managed Ubuntu minion and do some basic operations on i
     When I follow "Software" in the content area
     And I follow "Software Channels" in the content area
     And I wait until I do not see "Loading..." text
-    And I check radio button "Test Base Channel"
+    And I check radio button "Ubuntu AMD64 Channel"
```

Comment:
The new name of the channel will be `Test-Channel-Deb-AMD64` once #486 is merged.

Reply:
Okay, I will change it in the other tests aswell